data/competencies.yml

```diff
@@ -1,16 +1,292 @@
 
+- id: 0137fb68-807a-4971-8431-1b8bfd9a4bae
+  summary: Shadows during live incidents or joins debugging mob sessions
+  examples: []
+  supportingUrls: []
+  description: null
+  level: junior
+  area: technical
+  domain: null
+
+- id: 9098cc7f-a50a-48df-b84f-c4ee9fa47dbe
+  summary: Is able to explain why using CI/CD pipelines for automation is beneficial, and uses them for basic functions
+  examples:
+    - Restarts a broken build
+    - Is able to interpret CI output and articulate what is failing
+    - Can fix simple problems or asks for help
+  supportingUrls: []
+  description: null
+  level: junior
+  area: technical
+  domain: null
+
+- id: ae32dc05-3e53-47d2-a027-345eb42dffa3
+  summary: Occasionally uses monitoring tools to identify problems
+  examples:
+    - Occasionally uses Pingdom to identify problems
+    - Occasionally uses Grafana to identify problems
+  supportingUrls: []
+  description: null
+  level: junior
+  area: technical
+  domain: null
+
+- id: 70ae7051-3aed-42a0-be39-6c8a381eeaaf
+  summary: Writes pull requests that are helpful both to the reviewer and those debugging the the code later
+  examples:
+    - Writes PR descriptions that explain why the PR is needed
+    - Links to relevant Jira tickets in PRs
+    - Adds screenshots to show changes if applicable
+    - Provides context that they learned when researching a PR
+  supportingUrls: []
+  description: null
+  level: junior
+  area: communication
+  domain: null
+
+- id: 57c50830-f2ef-4c61-97fa-ec80b0624473
+  summary: Regularly reads peers' PRs and asks questions in order to learn
+  examples: []
+  supportingUrls: []
+  description: null
+  level: junior
+  area: communication
+  domain: null
+
+- id: 5d637365-7e38-432d-bcec-8652f06d8109
+  summary: Remembers to write more straightforward tickets and bug reports to track the work expected of them, perhaps with prompting and guidance
+  examples:
+    - Checks there is a corresponding ticket or bug report before starting any work, and flags it if not
+    - Writes reasonable "steps to reproduce" in a reasonably straightforward bug report when prompted
+    - Adds relevant links and screenshots to pages affected by a bug when prompted
+  supportingUrls: []
+  description: null
+  level: junior
+  area: communication
+  domain: null
+
+- id: dcfbf24d-04e2-4029-88a0-93479e4d04a8
+  summary: Receives feedback and acts upon it
+  examples:
+    - Implements requested changes on a pull request
+    - Responds to comments on a Google document
+    - If they receive feedback from the team that sometimes they don't seem well-prepared for morning stand-ups, takes 5 minutes before stand-ups to gather thoughts about what the status of their work is
+  supportingUrls: []
+  description: null
+  level: junior
+  area: communication
+  domain: null
+
+- id: c8d62965-dd71-4a90-83df-a4c924a40fd8
+  summary: Is able to recognise when they are having difficulties, provides information so that other people can help
```

Comment:
I wonder if this discriminates against  certain backgrounds or neurodiversities - it can take years to build up confidence, aptitudes and cultural habits to do this, and for junior people I think it's reasonable that the manager takes a lot of ownership of this growth

TBH, it's more the examples I'm nervous about than this general topic (which I think would be good to retain but in a softer way e.g. 'When prompted, is able to work with others to overcome any difficulties')

Reply:
I've discussed this one with @rowanmanning (and the others, too, but they were more straightforward).

This is a good question, but I think any changes would be guesswork without one or two specific examples of how the text here would lead to problems. As for the examples, they are only examples, not requirements, so thejy don't need to happen. And think the suggested text ("When prompted...") doesn't convey the concept of self-awareness, which is what this is about.

All that said, it could be more supportive, so I'll change the examples to imply more "these are ways you can get help" and less "these are things you must do".

Reply:
@dereknahman The [latest text is here](https://github.com/niksilver/engineering-progression/blob/007dfcbec2996a282c8798fb12c74054088e9538/data/competencies.yml#L161). I hope those examples are more constructive and less heavy/emergency-sounding.

Reply:
Nicely put @niksilver 👍